### PR TITLE
Feat/incident migration

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

### CODE REVIEW

File: packages/manager/apps/hub/src/dashboard/incident/migration/migration-warning/translations/Messages_de_DE.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "manager_hub_incident_migration_warning_replace": "Service replacement",
+  "manager_hub_incident_migration_warning_see_contracts": "View contracts",
+  "manager_hub_incident_migration_warning_cancel": "Cancel",
+  "manager_hub_incident_migration_warning_description": "o replace your service(s), you need to agree to the related contracts",
```
Comment (antleblanc) [inline packages/manager/apps/hub/src/dashboard/incident/migration/migration-warning/translations/Messages_de_DE.json:5]: Following translation seems incorrect (still in draft).

**Comment by marie-j** [comment]
I will clean history before merging

File: packages/manager/apps/hub/src/dashboard/incident/migration/routing.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        $http
+          .get(`/me/incident/${incidentName}/migrateServices`)
+          .then(({ data }) => data),
+      migrateServices: /* @ngInject */ ($http, incidentName) => (
+        dryRun,
+        serviceIds,
+      ) =>
+        $http
+          .post(`/me/incident/${incidentName}/migrateServices`, {
+            dryRun: true,
```
Comment (marie-j) [inline packages/manager/apps/hub/src/dashboard/incident/migration/routing.js:92]: Need to update this to pass params for QA

**Comment by marie-j** [comment]
Content might change

**Review by cbourgois — DISMISSED** [review]
👍

**Review by cbourgois — DISMISSED** [review]

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/apps/hub/src/dashboard/incident/migration/contracts/template.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<form name="contractsToAgree" novalidate>
+    <oui-modal
+        heading="{{:: 'manager_hub_incident_migration_contract' | translate}}"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/hub/src/dashboard/incident/migration/contracts/template.html:3]:
```suggestion
        heading="{{:: 'manager_hub_incident_migration_contract' | translate }}"
```

File: packages/manager/apps/hub/src/dashboard/incident/migration/migration-warning/template.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<oui-modal
+    primary-label="{{ 'manager_hub_incident_migration_warning_see_contracts' | translate }}"
+    secondary-label="{{ 'manager_hub_incident_migration_warning_cancel' | translate}}"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/hub/src/dashboard/incident/migration/migration-warning/template.html:3]:
```suggestion
    secondary-label="{{ 'manager_hub_incident_migration_warning_cancel' | translate }}"
```

File: packages/manager/apps/hub/src/dashboard/incident/migration/migration-warning/template.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    secondary-action="$ctrl.cancel()"
+    primary-action="$ctrl.confirm()"
+    on-dismiss="$ctrl.cancel()"
+    type="info"
+    heading="{{ :: 'manager_hub_incident_migration_warning_replace' | translate }}"
```
Comment (mohammed-zahaf) [inline packages/manager/apps/hub/src/dashboard/incident/migration/migration-warning/template.html:8]:
```suggestion
    heading="{{:: 'manager_hub_incident_migration_warning_replace' | translate }}"
```

**Review by cbourgois — DISMISSED** [review]

File: packages/manager/apps/hub/src/dashboard/incident/migration/contracts/template.html (hunk 3 above)
Comment (cbourgois) [reply packages/manager/apps/hub/src/dashboard/incident/migration/contracts/template.html:3]: Non-blocking, should have linting for that if we really want to format AngularJS mustache expression in HTML file.

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by cbourgois — DISMISSED** [review]

**Review by cbourgois — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**